lib/cereal/grpc/grpc.go

```diff
@@ -180,6 +180,22 @@ func (g *GrpcBackend) DequeueWorkflow(ctx context.Context, workflowNames []strin
 		return nil, nil, errors.New("unexpected")
 	}
 
+	tsProto := deq.GetEvent().GetEnqueuedAt()
+	ts := time.Time{}
+	if tsProto == nil {
+		// TODO(ssd) 2019-11-22: Determine what we want to do
```

Comment:
zero value seems reasonable

Reply:
the godoc could mention that its probably best to test for 0

Reply:
Which doc are you looking at?  I'm not seeing a function for the Timestamp type that does a zero test similar to `time.IsZero`.  However, it _appears_ that zero-values for both types round-trip correctly:

```
package main

import (
        "fmt"
        "time"

        "github.com/golang/protobuf/ptypes"
        tspb "github.com/golang/protobuf/ptypes/timestamp"
)

func main() {
        fmt.Println("Test time.Time{} round-trip")
        v0, err := ptypes.TimestampProto(time.Time{})
        fmt.Printf("ptypes.TimestampProto(time.Time{}) = %v (err: %v)\n", v0, err)
        v1, err := ptypes.Timestamp(v0)
        fmt.Printf("ptypes.Timestamp(%v) = %#v (err: %v) (isZero: %v)\n", v0, v1, err, v1.IsZero())
        fmt.Println("")

        fmt.Println("Test tspb.Timestamp{} round-trip")
        v3, err := ptypes.Timestamp(&tspb.Timestamp{})
        fmt.Printf("ptypes.Timestamp(&tspb.Timestamp{}) = %#v (err: %v)\n", v3, err)
        v4, err := ptypes.TimestampProto(v3)
        fmt.Printf("ptypes.TimestampProto(%#v) = %#v (err: %v)\n", v3, v4, err)
}
```

Reply:
i think what i was trying to say was that in our cereal godoc, we could say that its possible you get a zero value (for example in the multinode case)